.github/workflows/test-check.yaml

```diff
@@ -114,10 +114,14 @@ jobs:
       - name: "⚙️ Install dependencies"
         id: install
         run: pip3 install .[dev]
-      - name: "🔬 Running transformers tests[skipping over finetune]"
+      - name: "🔬 Running transformers tests"
         if: always() && steps.install.outcome == 'success'
         run: |
           pytest tests/llmcompressor/transformers/compression -v
```

Comment:
What about the tests under obcq/gptq/oneshot/sparsification?

Reply:
They're all running. Lines 117 onwards. Just separating them out into separate steps